api/.env

```diff
@@ -17,6 +17,7 @@ WEBSERVER_HOST={$WEB_PORT_8080_TCP_ADDR}
 WEBSERVER_PORT=8080
 
 AUTH_MODEL=App\Models\User
+AUTH_DRIVER=spira
```

Comment:
How is this driver name registered? Can we name it something more descriptive like `jwt`